src/ripple/protocol/impl/BuildInfo.cpp

```diff
@@ -148,6 +148,13 @@ getEncodedVersion()
     return cookie;
 }
 
+bool
+localVersionLow(std::uint64_t othersVersion)
+{
+    static std::uint64_t const mask = 0x0000ffffff000000;
+    return (getEncodedVersion() & mask) < (othersVersion & mask);
```

Comment:
While this is work currently, it's not guaranteed to in the future; we should fix it now. What you want is this:

```
bool isNewerVersion(std::uint64_t version)
{
    std::uint64_t constexrp mask = 0x183B000000000000;
    if ((version & mask) == mask)
        return version > getEncodedVersion();
    return false;
}
```

Reply:
Thanks, replaced this function.